.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 parameters:
   java-tag:
     type: string
-    default: "21.0.2"
+    default: "21.0.6"
```

Comment:
these newer containers and ubuntu still do not seem to have the new docker version (v26) surprisingly, but may as well upgrade anyway since they work